tests/test_intero.py

```diff
@@ -0,0 +1,22 @@
+'''Unit tests for the intero.py module.'''
+
+import pytest
+
+
+@pytest.fixture
+def mock_vim(mocker):
+    mock = mocker.MagicMock()
+    mocker.patch.dict('sys.modules', vim=mock)
+    return mock
+
+
+def test_stack_dirname(mock_vim):
+    from intero import stack_dirname
+    mock_vim.configure_mock(**{'eval.return_value': '/home/foobar'})
```

Comment:
It would also be nice to comment what argument we expect the function to be called with.